config/routes.rb

```diff
@@ -11,6 +11,24 @@
       match '/create' => 'users#create', via: [:post]
       match '/email_taken' => 'users#email_taken', via: [:get]
     end
+
+    scope '/lesson' do
+      match '/all' => 'lessons#all', via: [:get]
+      match '/create' => 'lessons#create', via: [:post]
```

Comment:
I added `/create` for readability, but noticed now that `POST` on `/` is what rails would do by default if we used `resources` so I think this should be changed